packages/patterns/chatbot-note-composed.tsx

```diff
@@ -175,9 +174,12 @@ export default recipe<ChatbotNoteInput, ChatbotNoteResult>(
   ({ title, messages }) => {
     const allCharms = schemaifyWish<MentionableCharm[]>("#allCharms", []);
     const index = schemaifyWish<BacklinksIndex>("/backlinksIndex", {
-      backlinks: {},
       mentionable: [],
     });
+    const mentionable = schemaifyWish<MentionableCharm[]>(
```

Comment:
Worth double-checking that this still does the right thing with AST transformation, since it's the derive invocation that needs the JSON schema and I'm not sure the compiler unrolls this enough.